src/index.html

```diff
@@ -1,13 +1,56 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./index.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="product-card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="product-image"
+        class="product-card__image"
+      >
+
+      <div class="product-card__product-name">
+        <p>Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</p>
+      </div>
+
+      <span class="product-card__product-code">
+        Код товара: 195434
+      </span>
+
+      <div class="product-rate">
+        <ul class="stars stars--4">
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+        </ul>
+
+        <div class="product-rate__reviews">
+          Отзывов: 5
+        </div>
+      </div>
+
+      <div class="product-card__price">
+        <p class="product-card__cost-label">Цена:</p>
+        <p class="product-card__cost">69 999 грн</p>
+      </div>
+
+      <a
+        href="#"
+        class="product-card__button-link"
+        data-qa="hover"
+      >
+        <button class="product-card__button">Купить</button>
```

Comment:
need put your link inside the button, than you will have cursor - pointer like the normal behavior of the button